R/tmb-methods.R

```diff
@@ -192,34 +194,163 @@ h_get_prediction_variance <- function(object, n_sim, tmb_data) {
 #' model.frame(object)
 #' model.frame(object, include = "subject_var")
 model.frame.mmrm_tmb <- function(formula, data, include = NULL, full, na.action = "na.omit", ...) { # nolint
-  include_choice <- c("subject_var", "visit_var", "group_var", "response_var")
+  # construct updated formula and data arguments
```

Comment:
Please use sentence style here and elsewhere, see https://openpharma.github.io/mmrm/latest-tag/CONTRIBUTING.html#comments
```suggestion
  # Construct updated formula and data arguments.
```